### ISSUE TITLE
Error loading discovery document - iOS

### ISSUE THREAD

**Issue by lorenh** [auth0/auth0-oidc-client-net#35]
We’ve received several reports from users on iOS that they’re now unable to log into our Xamarin.Forms app. Many of the reports are from users who have iPhone X and are running iOS 12.x

Based on our error logging we’ve found quite a few instances of two specific errors after calling Auth0Client.LoginAsync(extraParameters). The LoginResult.Error is “Error loading discovery document: Error connecting to https://{auth0tenant}.auth0.com/.well-known/openid-configuration: A task was canceled.” 
or
“Error loading discovery document: Error connecting to https://{auth0tenant}.auth0.com/.well-known/openid-configuration: An error occurred while sending the request”

Here’s a stack trace from one of these errors:

Message: Error loading discovery document: Error connecting to https://{auth0tenant}.auth0.com/.well-known/openid-configuration: A task was canceled.
IdentityModel.OidcClient.OidcClient.EnsureProviderInformationAsync() in <73424da05e224d8489755ed33593dddc#63812a54afbdd64580b3a5b121123909>:0
IdentityModel.OidcClient.OidcClient.EnsureConfigurationAsync() in <73424da05e224d8489755ed33593dddc#63812a54afbdd64580b3a5b121123909>:0
IdentityModel.OidcClient.OidcClient.LoginAsync() in <73424da05e224d8489755ed33593dddc#63812a54afbdd64580b3a5b121123909>:0
IdentityModel.OidcClient.OidcClient.LoginAsync() in <73424da05e224d8489755ed33593dddc#63812a54afbdd64580b3a5b121123909>:0
RiderXApp.Ios.LoginUIService.DisplayLoginUi() in <bc7f553aeaa44b3286b19cf4bbdfbc68#63812a54afbdd64580b3a5b121123909>:0
RiderXApp.Portable.LoginPageModel.DisplayLoginUi() in <3bb84c2a643e481e96115268db26fa74#63812a54afbdd64580b3a5b121123909>:0
Message: A task was canceled.
System.Net.Http.HttpClientHandler.SendAsync() in <6a949247cae943bba6c87f3db5f4b779#63812a54afbdd64580b3a5b121123909>:0
System.Net.Http.HttpClient.SendAsyncWorker() in <6a949247cae943bba6c87f3db5f4b779#63812a54afbdd64580b3a5b121123909>:0
IdentityModel.Client.DiscoveryClient.GetAsync() in <4b7ba16f640c48e1bc7f205c5de390e2#63812a54afbdd64580b3a5b121123909>:0

We’re using version 2.0.0 of the Auth0.OidcClient.iOS package from NuGet in our app.

We’re having a hard time understanding what would cause this issue and would appreciate any help.

**Comment by damieng** [comment]
Are you still experiencing this issue?

**Comment by lorenh** [comment]
I think we can close the issue.  We aren't sure what it was.  We have updated to the latest version of the packages and will be monitoring things for any changes.  We have scattered reports of iOS devices just not able to log in no matter what they try, so there are still some unsolved mysteries. Not sure if it's some Safari security or cookie settings, or privacy blockers or what, but fortunately it doesn't seem to be affecting the vast majority of our users and appears to be an edge case.
